### PR TITLE
Terraform 0.12.30 Upgrade

### PR DESCRIPTION
**Pull request by soutenniza**
## Changes proposed in this pull request:
- Pretty much a lot of format changes done by the upgrade tool.
- The main change is how lists are being done and used.
- Terraform version is 0.12.30
- AWS Provider version is 3.26
- Dev was upgraded and plan/apply before and after will do no infrastructure changes.

## security considerations
None

### CODE REVIEW

**Comment by mheadd** [comment]
UF1R

File: terraform/stacks/main/iam.tf [hunk 1]
```diff
@@ -1,289 +1,362 @@
...
+    module.bosh_compilation_role.role_name,
   ]
 }
 
 resource "aws_iam_policy_attachment" "blobstore_upstream" {
-  name = "${var.stack_description}-blobstore-upstream"
-  policy_arn = "${module.blobstore_upstream_policy.arn}"
+  name       = "${var.stack_description}-blobstore-upstream"
+  policy_arn = module.blobstore_upstream_policy.arn
+  # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
```
Comment (bengerman13) [inline terraform/stacks/main/iam.tf:201]: there's a TODO here - is it TODONE now?

File: terraform/stacks/main/stack.tf [hunk 2]
```diff
@@ -1,234 +1,251 @@
...
 resource "aws_lb" "main" {
   name    = "${var.stack_description}-main"
-  subnets = ["${module.stack.public_subnet_az1}", "${module.stack.public_subnet_az2}"]
-
-  security_groups = ["${var.force_restricted_network == "no" ?
-    module.stack.web_traffic_security_group :
-    module.stack.restricted_web_traffic_security_group}"]
+  subnets = [module.stack.public_subnet_az1, module.stack.public_subnet_az2]
+
+  # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
```
Comment (bengerman13) [inline terraform/stacks/main/stack.tf:45]: another TODO or TODONE

File: terraform/stacks/tooling/elb_uaa.tf [hunk 3]
```diff
@@ -1,46 +1,55 @@
 resource "aws_lb" "opsuaa" {
-  name = "${var.stack_description}-opsuaa"
-  subnets = ["${module.stack.public_subnet_az1}", "${module.stack.public_subnet_az2}"]
-  security_groups = ["${module.stack.restricted_web_traffic_security_group}"]
+  name    = "${var.stack_description}-opsuaa"
+  subnets = [module.stack.public_subnet_az1, module.stack.public_subnet_az2]
+  # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
```
Comment (bengerman13) [inline terraform/stacks/tooling/elb_uaa.tf:4]: TODO? TODONE? TODONT?

File: terraform/stacks/tooling/iam.tf [hunk 4]
```diff
@@ -131,62 +130,87 @@ module "concourse_iaas_worker_role" {
...
   ]
 }
 
 resource "aws_iam_policy_attachment" "bosh_compilation" {
   name       = "${var.stack_description}-bosh-compilation"
-  policy_arn = "${module.bosh_compilation_policy.arn}"
-
+  policy_arn = module.bosh_compilation_policy.arn
+
+  # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
```
Comment (bengerman13) [inline terraform/stacks/tooling/iam.tf:170]: more TODOs

**Review by mheadd — APPROVED** [review]
Thanks, @soutenniza!

**Pull request merged**